tcf

```diff
@@ -555,10 +527,74 @@ if __name__ == "__main__":
                     default = None, help = "Target's name or URL")
     ap.set_defaults(func = healthcheck)
 
+    tcfl.tc.target_ext_certs.cmdline_setup(arg_subparsers)
+
+
+    # advanced commands
+
+    ap = arg_subparsers.add_parser("cookies",
+                                   help = "Show logging cookies (to feed"
+                                   " into curl, etc) maybe only for one server")
+    ap.add_argument("target", metavar = "TARGET", action = "store",
+                    default = [], help = "Target name")
+    ap.add_argument("-c","--cookiejar", action = "store_true", default = False,
+                    help = "Print in cookiejar format"
+                    " (https://curl.se/docs/http-cookies.html)")
+    ap.add_argument("-j","--json", action = "store_true", default = False,
+                    help = "Print in JSON format")
+    ap.set_defaults(func = _cookies)
+
+    tcfl.tc.target_ext_users._cmdline_setup_advanced(arg_subparsers)
+
+    ap = arg_subparsers.add_parser("enable",
+                                   help = "Enable a disabled target")
+    ap.add_argument("target", metavar = "TARGET", action = "store",
+                    nargs = "+", default = None, help = "Target's name or URL")
+    ap.set_defaults(func = _target_enable)
+
+    ap = arg_subparsers.add_parser("disable",
+                                   help = "Disable an enabled target")
+    ap.add_argument("target", metavar = "TARGET", action = "store",
+                    nargs = "+", default = None, help = "Target's name or URL")
+    ap.add_argument("-r", "--reason", metavar = "REASON", action = "store",
+                    default = 'disabled by the administrator',
+                    help = "Reason why targets are disabled")
+    ap.set_defaults(func = _target_disable)
+
+
+    ap = arg_subparsers.add_parser(
+        "patch", help = "Store target information in the server")
+    ap.add_argument("target", metavar = "TARGET", action = "store",
+                    default = None, help = "Target's name")
+    ap.add_argument("data", metavar = "KEY=JSON-VALUE", nargs = "*",
+                    default = None, help = "Data items to store; if"
+                    " none, specify a JSON dictionary over stdin")
```

Comment:
what is the diff with `tcf property-set`?

Reply:
patch allows you to specify a whole json with multiple fields; property-set is just one property and value